config-paths.js

```diff
@@ -0,0 +1,23 @@
+const { app } = require('electron');
+const { homedir } = require('os');
+const { resolve } = require('path');
+const { existsSync } = require('fs');
+
+const home = homedir();
+const root = resolve(app.getPath('appData'), 'HyperTerm');
+const legacyConfig = resolve(home, './.hyperterm.js');
```

Comment:
Another nitpick, prepending `./` to all the paths are moot. `const legacyConfig = resolve(home, '.hyperterm.js');` would be better. `path.join()` would be a better option here, since `home` is already an absolute path, so you can just join them, instead of resolving.